README.md

```diff
@@ -158,6 +159,14 @@ Store a [CAR](https://ipld.io/specs/transport/car/carv1/) file to web3.storage.
 
 ### `w3 can store ls`
 
+List CARs in the current space.
```

Comment:
Is the idea that these will list a page of results and `w3 ls` would list _everything_ automatically?

Reply:
Curious how we expose the cursor for getting the next page...

Reply:
> Is the idea that these will list a page of results and w3 ls would list everything automatically?

Yep - I didn't want to mess with the semantics of `w3 ls` too much in this work and just needed the lower level `can` commands - per your other comment I'm going to roll back the one change I did make to `w3 ls` because it doesn't make sense

> Curious how we expose the cursor for getting the next page...

Yea this was really the entire reason I added `--raw` - the existing default, `--json` and `--shards` options return lists of results and don't really have a place to return metadata like paging cursors. That's definitely a design decision we could revisit, but to be imho it's the right design.

Practically, because (for now at least) cursors are actually just the CIDs of the first and last items in a list, users at the command line can just use the results they get from the results-level formatting options, so `--raw` is likely to be used primarily by scripts and other users that need to rely on the "correct" semantics of opaque cursors.